Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -567,20 +572,38 @@ def create_incident(notable_data, occurred, mapper: UserMappingObject):
                 labels.append({'type': key, 'value': value})
         if demisto.get(notable_data, 'security_domain'):
             labels.append({'type': 'security_domain', 'value': notable_data["security_domain"]})
+        if demisto.get(notable_data, 'comment'):
+            comments = argToList(notable_data.get('comment', []))
+            reviewers = argToList(notable_data.get('reviewer', []))
+            review_times = argToList(notable_data.get('review_time', []))
+
+            demisto.debug(f"data to update comment= {comments}, review=  {reviewers}, time= {review_times}")
+            for comment, reviewer, review_time in zip(comments, reviewers, review_times):
```

Comment:
What will be the behavior in case of an `User_1` added a comment in `09:00AM` but after that `User_2` change something else in `10:00AM`
e.g. `comment=['comment_1]' reviewer=[User_2, User1], review_time=['10:00AM', '09:00AM']`?

Reply:
fix that behavior